torch/_export/exported_program.py

```diff
@@ -337,7 +337,7 @@ def _process_constraints(
         val = placeholder_nodes[input_dim.input_name].meta["val"]
         assert isinstance(val, FakeTensor)
         symint = val.shape[input_dim.dim]
-        assert isinstance(symint, SymInt)
+        assert isinstance(symint, SymInt), f"Expected SymInt but got {symint}: {type(symint)}"
```

Comment:
@avikchaudhuri does this mean we still ran into assertion failure?

Reply:
Only if you try passing a 0 or 1 dimensional tensor in KJT, yes. In all other cases this assert should not fail.

Reply:
For KJT you get an int that is 0 or 1.

Reply:
ah I see, it makes sense now.